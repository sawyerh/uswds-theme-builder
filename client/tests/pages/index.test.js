import Page from "pages/index";
import { render, fireEvent, waitFor, screen } from "@testing-library/react";

describe("Index page", () => {
  it("shows Editor panel by default", () => {
    render(<Page />);
  });
});
