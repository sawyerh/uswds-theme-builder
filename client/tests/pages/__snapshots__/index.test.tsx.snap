// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders inputs for Sass color variables 1`] = `
Array [
  "$theme-color-base-lightest",
  "$theme-color-base-lighter",
  "$theme-color-base-light",
  "$theme-color-base",
  "$theme-color-base-dark",
  "$theme-color-base-darker",
  "$theme-color-base-darkest",
  "$theme-color-base-ink",
  "$theme-color-primary-lightest",
  "$theme-color-primary-lighter",
  "$theme-color-primary-light",
  "$theme-color-primary",
  "$theme-color-primary-vivid",
  "$theme-color-primary-dark",
  "$theme-color-primary-darker",
  "$theme-color-primary-darkest",
  "$theme-color-secondary-lightest",
  "$theme-color-secondary-lighter",
  "$theme-color-secondary-light",
  "$theme-color-secondary",
  "$theme-color-secondary-vivid",
  "$theme-color-secondary-dark",
  "$theme-color-secondary-darker",
  "$theme-color-secondary-darkest",
  "$theme-color-accent-cool-lightest",
  "$theme-color-accent-cool-lighter",
  "$theme-color-accent-cool-light",
  "$theme-color-accent-cool",
  "$theme-color-accent-cool-dark",
  "$theme-color-accent-cool-darker",
  "$theme-color-accent-cool-darkest",
  "$theme-color-accent-warm-lightest",
  "$theme-color-accent-warm-lighter",
  "$theme-color-accent-warm-light",
  "$theme-color-accent-warm",
  "$theme-color-accent-warm-dark",
  "$theme-color-accent-warm-darker",
  "$theme-color-accent-warm-darkest",
  "$theme-color-error-lighter",
  "$theme-color-error-light",
  "$theme-color-error",
  "$theme-color-error-dark",
  "$theme-color-error-darker",
  "$theme-color-warning-lighter",
  "$theme-color-warning-light",
  "$theme-color-warning",
  "$theme-color-warning-dark",
  "$theme-color-warning-darker",
  "$theme-color-success-lighter",
  "$theme-color-success-light",
  "$theme-color-success",
  "$theme-color-success-dark",
  "$theme-color-success-darker",
  "$theme-color-info-lighter",
  "$theme-color-info-light",
  "$theme-color-info",
  "$theme-color-info-dark",
  "$theme-color-info-darker",
  "$theme-color-emergency",
  "$theme-color-emergency-dark",
  "$theme-color-disabled-light",
  "$theme-color-disabled",
  "$theme-color-disabled-dark",
]
`;
